builder/llpcBuilderRecorder.h

```diff
@@ -242,6 +243,13 @@ class BuilderRecorder final : public Builder, BuilderRecorderMetadataKinds
     // the given cube map texture coordinates.
     Value* CreateCubeFaceIndex(Value* pCoord, const Twine& instName = "") override final;
 
+
+    // Create scalar or vector FP truncate operation with rounding mode.
+    Value* CreateFPTruncWithRounding(Value*                                pValue,
```

Comment:
CreateFPTruncWithRounding -> CreateFpTruncWithRounding

Reply:
Yes, make this change unless we just overwrite LLVM function. These are two different functions actually. LLVM's CreateFPTrunc is a wrapper to new FPTrunc instruction and our function is to do complex emulation to respect rounding mode.